charts/hono/Chart.yaml

```diff
@@ -15,7 +15,7 @@ name: hono
 description: |
   Eclipse Hono™ provides remote service interfaces for connecting large numbers of IoT devices to a back end and
   interacting with them in a uniform way regardless of the device communication protocol.
-version: 2.1.5
```

Comment:
Regarding cloud2edge compatibility, we have 2 options here:
- use version `3.0.0` here. That means the current  cloud2edge chart won't reference it (`^2.1.0` used there now).
- use version `2.2.0` here and first create a new cloud2edge chart version that only changes the Hono dependency requirement from `^2.1.0` to `~2.1.0`.

I would tend towards the 2nd option here. Having just the patch-level version range in the cloud2edge chart offers more flexibility regarding Hono chart changes affecting the cloud2edge chart, without having to do a major version jump of the Hono chart each time.
@sophokles73 WDYT?

Reply:
I also gravitate more towared the second option here.